app/json_data.py

```diff
@@ -0,0 +1,35 @@
+import json
+import math
+
+from app.shop import Shop
+from app.customer import Customer
+
+
+def create_elements() -> tuple:
+    with open("app/config.json") as file:
+        data = json.load(file)
+    fuel_price = data["FUEL_PRICE"]
+    customers = Customer.create_customers(data["customers"])
+    shops = Shop.create_shops(data["shops"])
+    return fuel_price, customers, shops
+
+
+def distance(point1: list, point2: list) -> float:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]